generators/kubernetes/templates/registry/_jhipster-registry.yml

```diff
@@ -84,17 +84,13 @@ spec:
         # Registry configuration
         - name: SPRING_PROFILES_ACTIVE
           value: prod,native
-        - name: SECURITY_USER_PASSWORD
+        - name: JHIPSTER_REGISTRY_PASSWORD
           valueFrom:
             secretKeyRef:
               name: registry-secret
               key: registry-admin-password
         - name: JHIPSTER_SECURITY_AUTHENTICATION_JWT_SECRET
           value: my-secret-token-to-change-in-production
-        - name: EUREKA_CLIENT_FETCH_REGISTRY
-          value: 'true'
-        - name: EUREKA_CLIENT_REGISTER_WITH_EUREKA
-          value: 'true'
```

Comment:
I think these properties are required and need to be set 'true'. This is required to avoid DiscoveryComposite - eureka 'Unknown' in the console. Though you have defined the individual instances to be discovered in the 'command' tag, without these properties set you will not get the status 'UP' in the registry console. I had the same issue when I tested it last time in K8s couple of months ago. Did you manage to check the console for status after removing these properties?
[More info](https://github.com/jhipster/jhipster-registry/issues/69) can be found in this thread